src/main/java/com/spotinst/sdkjava/model/api/admin/account/ApiOrganizationUserGroups.java

```diff
@@ -0,0 +1,122 @@
+package com.spotinst.sdkjava.model.api.admin.account;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiOrganizationUserGroups implements IPartialUpdateEntity {
```

Comment:
Didn't get this class.

Reply:
This class is to get all groups in the organization.